client/src/style.css

```diff
@@ -2205,6 +2205,29 @@ ol ol):not(:where([class~=not-prose] *)) {
   
 }
 
+.focus-visible-button {
+  background-color: transparent;
+  transition: background-color 0.2s;
+}
+
+* {
```

Comment:
is this right? every element should have this?

Reply:
Applying a transition to every element using the universal selector (*) is generally considered bad practice because:

1. It can significantly impact performance, especially on large websites.
2. It may cause unexpected and unwanted transitions on elements.
3. It complicates CSS specificity and makes overriding styles more difficult.
4. It can lead to maintenance issues and make debugging more challenging.
5. It might create accessibility problems for users sensitive to motion.

Instead, it's better to apply transitions selectively to specific elements or classes where they are actually needed.

Reply:
yes, I think that adding a transition color to all the elements will reduce the className lenght and make everything smoother without adding `transition-color` and `duration-200`